src/sim/mockSim.js

```diff
@@ -33,3 +33,19 @@ let gen = nextInstructionGen();
 exports.mockStepForward = () => {
     return gen.next().value;
 };
+
+exports.mockStepBackward = () => {
+    return 'stepping back';
```

Comment:
can we make this similar to mockStepForward() and have it return gen.prev().value? I think that would help for creating/debugging the front end stuff

Reply:
True. I thought of that but I wasn’t exactly sure if it was as simple as that. I’m gonna pose the question in slack and see what ideas they have